src/lib/components/home/artwork_rails/artwork_rail.tsx

```diff
@@ -55,15 +58,17 @@ export class ArtworkRail extends React.Component<Props & RelayPropsWorkaround, S
       expanded: false,
       gridHeight: 0,
       loadFailed: false,
+      didPerformFetch: false,
     }
   }
 
   componentDidMount() {
     if (this.props.relay) {
       this.props.relay.setVariables({ fetchContent: true }, readyState => {
-        if (readyState.error) {
-          this.setState({ loadFailed: true })
-        }
+        this.setState({
+          didPerformFetch: readyState.done,
```

Comment:
`readyState.done` is false initially. It gets set to true with all the data requirements are met. Should I check the "ready" state instead?

https://facebook.github.io/relay/docs/guides-ready-state.html#fetching-data-from-the-server